### PR TITLE
added class for thank you title, also updated remaining eff-red-butto…

### LINKED ISSUE #688: Center "Thank You" text on action thank you pages
Center the "Thank You" text on the action thank you pages 
e.g., https://act.staging.eff.org/action/tell-the-senate-to-restore-full-net-neutrality-protections
Screen Shot 2019-09-09 at 10 39 16 AM

### LINKED ISSUE #690: Update donate button to match others
The donate button should match the other buttons, like the "Find Your reps" button with white text

Screen Shot 2019-09-09 at 11 06 58 AM

### PR DESCRIPTION
**Pull request by lschatzkin**
closes #688 
also closes #690

### CODE REVIEW

**Comment by esoterik** [comment]
I think this happened somewhere else, but the tweet "go back" text styling broke, and it would back sense to fix it here I think:
image

I think it could either look like a link or a wide, blue button (like the test button for admins on congress forms), whichever you think is best!

**Comment by lschatzkin** [comment]
Okay, I'll fix this now.

**Comment by lschatzkin** [comment]
Um, how do I make this text appear? I'm testing a tweet action but I don't see this.

**Comment by esoterik** [comment]
it only shows on tweet actions with > 1 target -- I've been using `action/illinois-say-no-to-warrantless-drone-surveillance` for testing

**Comment by lschatzkin** [comment]
This isn't actually a button that does anything, so I made it look more like text.

**Comment by esoterik** [comment]
good to merge after conflicts are resolved!

File: app/assets/stylesheets/action_page.scss [hunk 1]
```diff
@@ -373,21 +377,30 @@
...
+      font-size: $base-font-size * 0.8;
+      margin: 1rem auto;
+      max-width: 100%;
+      text-transform: none;
       &:hover {
-        cursor: pointer;
+        background: $gray-light;
+        border-bottom: 0;
+        color: $charcoal;
+        cursor: default;
```
Comment (esoterik) [inline app/assets/stylesheets/action_page.scss]: can the cursor be a pointer again? it's not super obvious that it's clickable to me

**Comment by esoterik** [comment]
I'm still seeing text alignment issues with the "go back" link
image

File: app/assets/stylesheets/action_page.scss (hunk 1 above)
Comment (lschatzkin) [reply app/assets/stylesheets/action_page.scss]: No, that's the thing, it's not clickable.

Comment (esoterik) [reply app/assets/stylesheets/action_page.scss]: clicking it brings you back to the take action tool though

Comment (lschatzkin) [reply app/assets/stylesheets/action_page.scss]: I've lost my damn mind.

**Comment by lschatzkin** [comment]
> I'm still seeing text alignment issues with the "go back" link
> image

Is this still happening? I'm seeing this:

**Comment by lschatzkin** [comment]
Screenshot from 2019-09-11 09-53-19

**Comment by esoterik** [comment]
:sparkles: merging!

**Pull request merged**